cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -544,4 +544,48 @@ describe('Dashboards', () => {
       })
     })
   })
+
+  before('Clear downloads folder', () => {
+    cy.exec('rm cypress/downloads/*', {log: true, failOnNonZeroExit: false})
```

Comment:
I believe we can move this into the `beforeEach` since we want this directory to be clean on every test

Reply:
removal of the file has been moved into beforeEach